config/options

```diff
@@ -96,6 +96,10 @@ if [ -f "${HOME}/.libreelec/options" ]; then
   . "${HOME}/.libreelec/options"
 fi
 
+# set addon paths
+ADDON_PATH="$ADDON_VERSION/$ADDON_PROJECT/$TARGET_ARCH"
+ADDON_URL="$ADDON_SERVER_URL/$ADDON_PATH"
```

Comment:
This should probably be moved earlier so that we can override the `ADDON_URL` in `.libreelec/options`